development-defaults.ini

```diff
@@ -68,21 +72,53 @@ attendee_badge = 3000, 29999
 one_day_badge = 30000, 39999
 
 
+[age_groups]
+
+[[under_7]]
+desc            = "Under 7"
+min_age         = 0
+max_age         = 7
+discount        = 999
+can_volunteer   = False
+consent_form    = True
+wristband_color = "red"
+
+[[under_13]]
+desc            = "Between 7 and 13"
+min_age         = 7
+max_age         = 12
+discount        = 10
+can_volunteer   = False
+consent_form    = True
+wristband_color = "red"
+
+[[under_18]]
+desc            = "Between 13 and 18"
+min_age         = 13
+max_age         = 18
+can_volunteer   = False
+consent_form    = True
+wristband_color = "red"
+
+[[under_21]]
+desc            = "Between 18 and 21"
+min_age         = 18
+max_age         = 20
+wristband_color = "blue"
+
+[[over_21]]
+desc            = "21 or older"
+min_age         = 21
+max_age         = 99
+wristband_color = "green"
+
 
 [integer_enums]
 size_unknown = -1
 no_shirt     = 0
 
 [[donation_tier]]
```

Comment:
I've been trying to move a lot of the MAGFest-specific stuff out of ``development-defaults.ini`` but a lot of it is still here.  When https://github.com/magfest/ubersystem-puppet/issues/28 is complete it'll make it a lot easier to have example configs for different events which all compile into ``development.ini``.